calculator.js

```diff
@@ -1,5 +1,45 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
-}
+
+const prompt = require('prompt');
+
+const calculateUserInput = function(error, input) {
+  console.log(input);
+  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', input);
+
+
+  calculate(input);
+}  
+
+prompt.start();
+
+prompt.get(['num1','num2','operation'], calculateUserInput);
+
+function calculate(input) {
+  if (isNaN(input.num1) || isNaN(input.num2) || typeof input.num1 === "boolean" || typeof input.num2 === "boolean") throw "invalid input";
+
+  if(input.operation === "add" || input.operation === "+"){
+
+    let add = parseInt(input.num1) + parseInt(input.num2);
+    console.log(input.num1 + " + " + input.num2 + " = " + add);
+
+  }else if(input.operation === "subtract" || input.operation === "-"){
```

Comment:
extremely minor critique, just some spacing.
```suggestion
  } else if(input.operation === "subtract" || input.operation === "-") {
```